### PR TITLE
deploy docs to JuliaLang/docs.julialang.org

### PR DESCRIPTION
**Pull request by fredrikekre**
Deploys too https://github.com/JuliaLang/docs.julialang.org instead of bloating this repo

cc @mortenpi

### CODE REVIEW

File: doc/Manifest.toml [hunk 1]
```diff
@@ -72,7 +74,7 @@ deps = ["Distributed", "InteractiveUtils", "Logging", "Random"]
 uuid = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [[UUIDs]]
-deps = ["Random", "SHA"]
+deps = ["Random"]
```
Comment (fredrikekre) [inline doc/Manifest.toml]: What happened here? @KristofferC ?

Comment (StefanKarpinski) [reply doc/Manifest.toml]: Different Julia versions.

Comment (fredrikekre) [reply doc/Manifest.toml]: In which Julia version does `UUIDs` not depend on `SHA` though?

**Comment by fredrikekre** [comment]
For anyone interested this reduced the `JuliaLang/julia` repo with ~300MiB (65%) so I think it was well worth it.

**Comment by StefanKarpinski** [comment]
Absolutely worth it—thanks for persisting with that!

**Pull request merged**